test/fuzz/lua_fuzzer/lua_grammar.proto

```diff
@@ -0,0 +1,310 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+/* Grammar is for Lua 5.1 */
+syntax = "proto2";
+package lua_grammar;
+
+
+message LuaBlock {
+    required Chunk cur_chunk = 1;
+}
+
+message Chunk {
+    repeated StatementWithSemicolon stat = 1;
+    optional LastStatementWithSemicolon last = 2;
+}
+
+message StatementWithSemicolon {
+    required Statement stat = 1;
+    optional bool semicolon = 2;
+}
+
+message LastStatementWithSemicolon {
+    required LastStatement stat = 1;
+    optional bool semicolon = 2;
+}
+
+message Statement {
+    oneof stat_oneof {
```

Comment:
I decided not to encapsulate all clauses of Statement inside it, because then there will be nested messages depth >= 3 and I think this will be worse for readability and Statement message will be overloaded with other messages.